### ISSUE TITLE
Package requires nova <2.2

### ISSUE THREAD

**Issue by CitizenBeta** [chris-ware/nova-breadcrumbs#46]
I think this package is preventing my nova from updating. It says ChrisWare\NovaBreadcrumbs requires <2.2

**Comment by CitizenBeta** [comment]
Confirmed. As soon as I removed this package, nova was able to update to 2.3. It would be great if you could set the dependency for this to be ^2.0 instead of <2.2. Thanks so much for this package <3

**Comment by flxsource** [comment]
As stated in the read me:

> As new versions of Nova are released, if changes are made to any of the views that this package has to overwrite, a new version will need to be released. Composer version constraints should prevent installs on versions that aren't supported.

If the current supported version isn’t updated quickly enough for you, feel free to fork the repo and change the constraints yourself.

**Comment by CitizenBeta** [comment]
I'm actually not sure how to do that. Rather than forking, would it be more helpful for people like me to test on the latest version and if it works, do a pull request? What's the best way to help?

**Comment by chris-ware** [comment]
The limit is via composer itself, so it's not a case of testing it on the newest version as it isn't possible. The reason this constraint is in place is explained in the readme, as @flxsource has explained above. I recompile as quickly as I can between releases, but with updates happening fairly often at the moment, finding the time is hard. 

I have submitted an issue to the laravel-nova/issues repo https://github.com/laravel/nova-issues/issues/1875 which will either remove the need for recompiling per new version or remove the need for the package entirely, depending on how Laravel decide to implement the request.

I've just tagged 1.2.5 for release, which will now support 2.3.0.
